Office 365/o365/_meta/fields.yml

```diff
@@ -2,119 +2,94 @@ action.properties:
   description: A list of objects describing the action
   name: action.properties
   type: object
-
 action.target:
   description: The target of the action
   name: action.target
   type: keyword
-
+data.user.email:
+  description: The email of the user who triggered the alert
+  name: data.user.email
+  type: keyword
```

Comment:
`user.email` seems to be a more suitable field. The name of the field doesn't match the convention followed for Office365 custom field (ie the prefix should be `office365` and not `data`)